### ISSUE TITLE
Mount failed and error

### ISSUE THREAD

**Issue by artxty2** [master131/iFakeLocation#62]
Hi Master131,

I got this message "Mount failed with status: N/A and error: ImageMountFailed" only when i connect iphone with ios 15.1, but there is no problem when i connect it with an ipad with ios 15.0.2.

This scenario is same when i used the version 1.5 and version 1.6

btw i am using macos. i have also installed .NET 5.0 Runtime.

Thank you for creating this ifakelocation and i really appreciate all your hard work.

**Comment by master131** [comment]
Just to verify, the iPhone is trusted with the Mac and unlocked when you tried this?

**Comment by artxty2** [comment]
Hi Master131,
yes the iphone can be read by your program. or is there something i need to click to make it trusted by the mac?
i can see the iphone from the finder and the music app on the mac.
thank you-alvin


    On Tuesday, 30 November 2021, 04:59:27 pm AEDT, master131 ***@***.***> wrote:  
 
 


Just to verify, the iPhone is trusted with the Mac and unlocked when you tried this?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by master131** [comment]
It's honestly hard to say, I just tested it myself with an iPad running iOS 15.1 and it works without issues. Are you using a newer iPhone - what model is it?

**Comment by artxty2** [comment]
hi master131.
yes its a bit strange indeed, it works on ipad, but not on the iphone, i have iphone 11 pro.i had this problem after updating to the latest ios 15.1
i will try and make a video for both the ipad and iphone.
thank youalvin



Sent from Yahoo Mail for iPhone


On Friday, December 3, 2021, 8:55 pm, master131 ***@***.***> wrote:




It's honestly hard to say, I just tested it myself with an iPad running iOS 15.1 and it works without issues. Are you using a newer iPhone - what model is it?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by master131** [comment]
This might sound silly, but have you tried rebooting the phone since running into this issue?

**Comment by artxty2** [comment]
Hi master131,
I followed your suggestion, after rebooting the iphone it worked!
I tried on both Safari and Firefox browser and it works.
Thank you and I really appreciate your help. :)
kind regards,
alvin
    On Saturday, 4 December 2021, 10:02:53 am AEDT, master131 ***@***.***> wrote:  
 
 


This might sound silly, but have you tried rebooting the phone since running into this issue?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
